kodein-android/src/main/java/com/github/salomonbrys/kodein/android/KodeinAndroidComponents.kt

```diff
@@ -302,14 +308,16 @@ interface FragmentInjector : AndroidInjector<Fragment, AndroidScope<Fragment>> {
             activity?.let {
                 bindErased<LayoutInflater>(ACTIVITY_LAYOUT_INFLATER, overrides = true) with erasedInstance(activity.layoutInflater)
             }
-            bindErased<Fragment>() with erasedInstance(kodeinComponent)
+            bindErased<Fragment>(overrides = parentFragment != null) with erasedInstance(kodeinComponent)
```

Comment:
There should probably be a tagged binding to retrieve the parent if it is not null.